src/EditorFeatures/VisualBasicTest/Formatting/Indentation/SmartIndenterTests.vb

```diff
@@ -2904,6 +2904,8 @@ End Class
                             Return p
                         End Function)
 
+                WpfTestCase.RequireWpfFact("Test helper creates mocks of ITextView")
```

Comment:
Mocks still load the assembly, hence possibly invoke static ctors, hence yeah probably need `WpfFact`